Flaky TestIngester_PrepareInstanceRingDownscaleHandler

```
--- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler (3.00s)
    --- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only (0.00s)
        downscale_test.go:59: 
            	Error Trace:	/__w/mimir/mimir/pkg/ingester/downscale_test.go:59
            	            				/__w/mimir/mimir/vendor/github.com/grafana/dskit/test/poll.go:17
            	            				/__w/mimir/mimir/pkg/ingester/downscale_test.go:57
            	Error:      	Received unexpected error:
            	            	instance not found in the ring
            	            	github.com/grafana/dskit/ring.init
            	            		<autogenerated>:1
            	            	runtime.doInit1
            	            		/usr/local/go/src/runtime/proc.go:[72](https://github.com/grafana/mimir/actions/runs/11141470952/job/30962417169?pr=9493#step:8:73)90
            	            	runtime.doInit
            	            		/usr/local/go/src/runtime/proc.go:7257
            	            	runtime.main
            	            		/usr/local/go/src/runtime/proc.go:254
            	            	runtime.goexit
            	            		/usr/local/go/src/runtime/asm_amd64.s:1700
            	Test:       	TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only
level=info msg="uploading new block to long-term storage" block=00000000010000000000000000
level=debug msg="uploaded file" from=/tmp/TestShipper_DeceivingUploadErrors2045435997/001/00000000010000000000000000/index dst=00000000010000000000000000/index bucket="fs: /tmp/TestShipper_DeceivingUploadErrors2045435997/002"
level=error msg="uploading new block to long-term storage failed" block=00000000010000000000000000 err="upload meta file: base name matches, will fail upload"
FAIL
```

another flake

```
--- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler (3.00s)
    --- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only (0.01s)
        downscale_test.go:59: 
            	Error Trace:	/__w/mimir/mimir/pkg/ingester/downscale_test.go:59
            	            				/__w/mimir/mimir/vendor/github.com/grafana/dskit/test/poll.go:17
            	            				/__w/mimir/mimir/pkg/ingester/downscale_test.go:57
            	Error:      	Received unexpected error:
            	            	instance not found in the ring
            	            	github.com/grafana/dskit/ring.init
            	            		<autogenerated>:1
            	            	runtime.doInit1
            	            		/usr/local/go/src/runtime/proc.go:7290
            	            	runtime.doInit
            	            		/usr/local/go/src/runtime/proc.go:7257
            	            	runtime.main
            	            		/usr/local/go/src/runtime/proc.go:254
            	            	runtime.goexit
            	            		/usr/local/go/src/runtime/asm_amd64.s:1700
            	Test:       	TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only
level=info msg="uploading new block to long-term storage" block=00000000010000000000000000
level=debug msg="uploaded file" from=/tmp/TestShipper_DeceivingUploadErrors1375604059/001/00000000010000000000000000/index dst=00000000010000000000000000/index bucket="fs: /tmp/TestShipper_DeceivingUploadErrors1375604059/002"
level=error msg="uploading new block to long-term storage failed" block=00000000010000000000000000 err="upload meta file: base name matches, will fail upload"
FAIL
```

https://github.com/grafana/mimir/actions/runs/11383313024/job/31668680547?pr=9655
```
--- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler (3.00s)
    --- FAIL: TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only (0.00s)
        downscale_test.go:59: 
            	Error Trace:	/__w/mimir/mimir/pkg/ingester/downscale_test.go:59
            	            				/__w/mimir/mimir/vendor/github.com/grafana/dskit/test/poll.go:17
            	            				/__w/mimir/mimir/pkg/ingester/downscale_test.go:57
            	Error:      	Received unexpected error:
            	            	instance not found in the ring
            	            	github.com/grafana/dskit/ring.init
            	            		<autogenerated>:1
            	            	runtime.doInit1
            	            		/usr/local/go/src/runtime/proc.go:[72](https://github.com/grafana/mimir/actions/runs/11383313024/job/31668680547?pr=9655#step:8:73)90
            	            	runtime.doInit
            	            		/usr/local/go/src/runtime/proc.go:7257
            	            	runtime.main
            	            		/usr/local/go/src/runtime/proc.go:254
            	            	runtime.goexit
            	            		/usr/local/go/src/runtime/asm_amd64.s:1700
            	Test:       	TestIngester_PrepareInstanceRingDownscaleHandler/POST_request_should_switch_the_instance_ring_entry_to_read-only
level=info msg="uploading new block to long-term storage" block=00000000010000000000000000
level=debug msg="uploaded file" from=/tmp/TestShipper_DeceivingUploadErrors3861206[84](https://github.com/grafana/mimir/actions/runs/11383313024/job/31668680547?pr=9655#step:8:85)4/001/00000000010000000000000000/index dst=00000000010000000000000000/index bucket="fs: /tmp/TestShipper_DeceivingUploadErrors3[86](https://github.com/grafana/mimir/actions/runs/11383313024/job/31668680547?pr=9655#step:8:87)1206844/002"
level=error msg="uploading new block to long-term storage failed" block=00000000010000000000000000 err="upload meta file: base name matches, will fail upload"
```

--- a/pkg/ingester/downscale_test.go
+++ b/pkg/ingester/downscale_test.go
@@ -54,7 +54,7 @@ func TestIngester_PrepareInstanceRingDownscaleHandler(t *testing.T) {
 		ingester, r := setup(true)
 
 		// Pre-condition: entry is not read-only.
-		test.Poll(t, 5*time.Second, false, func() interface{} {
+		test.Poll(t, 10*time.Second, false, func() interface{} {
 			inst, err := r.GetInstance(ingester.lifecycler.ID)
 			require.NoError(t, err)
 			return inst.ReadOnly
@@ -69,7 +69,7 @@ func TestIngester_PrepareInstanceRingDownscaleHandler(t *testing.T) {
 		require.InDelta(t, time.Now().Unix(), resp.Timestamp, 10)
 
 		// Post-condition: entry is read only.
-		test.Poll(t, 5*time.Second, true, func() interface{} {
+		test.Poll(t, 10*time.Second, true, func() interface{} {
 			inst, err := r.GetInstance(ingester.lifecycler.ID)
 			require.NoError(t, err)
 			return inst.ReadOnly && inst.ReadOnlyUpdatedTimestamp == resp.Timestamp
@@ -95,7 +95,7 @@ func TestIngester_PrepareInstanceRingDownscaleHandler(t *testing.T) {
 		// Switch entry to read-only.
 		ingester.PrepareInstanceRingDownscaleHandler(res, httptest.NewRequest(http.MethodPost, target, nil))
 		require.Equal(t, http.StatusOK, res.Code)
-		test.Poll(t, 5*time.Second, true, func() interface{} {
+		test.Poll(t, 10*time.Second, true, func() interface{} {
 			inst, err := r.GetInstance(ingester.lifecycler.ID)
 			require.NoError(t, err)
 			return inst.ReadOnly
@@ -111,7 +111,7 @@ func TestIngester_PrepareInstanceRingDownscaleHandler(t *testing.T) {
 		require.Equal(t, int64(0), resp.Timestamp)
 
 		// Post-condition: entry is not read only.
-		test.Poll(t, 5*time.Second, false, func() interface{} {
+		test.Poll(t, 10*time.Second, false, func() interface{} {
 			inst, err := r.GetInstance(ingester.lifecycler.ID)
 			require.NoError(t, err)
 			return inst.ReadOnly

